examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -42,7 +47,9 @@ def forward(self, x):
         return output
 
 
-# NEW: Modify function header to include core_context for metric reporting.
+# Docs snippet start after: include core context
```

Comment:
this does not appear to be used

Reply:
side note, this snippet about `include core context` should really only include the `det train(...)` line, and nothing else.  It's about adding new args to this function signature.

Reply:
removed the docs snippet comment